### PR TITLE
Add favicon.ico

### PR DESCRIPTION
**Pull request by abitrolly**
I don't see it on localhost, so some testing would be welcome.

### CODE REVIEW

File: filmfest/static/logo.svg [hunk 1]
```diff
@@ -31,7 +32,7 @@
      showgrid="true"
      inkscape:document-units="px"
      inkscape:grid-bbox="true"
-     inkscape:window-width="1364"
+     inkscape:window-width="1280"
      inkscape:window-height="747"
```
Comment (staaas) [inline filmfest/static/logo.svg:36]: height and width should have the same value, shouldn't they?

**Comment by staaas** [comment]
Favicon looks good on my machine:

favicon_screenshot

Logo also looks fine:

logo_screenshot

I've noticed an issue with logo layout on a small screen (window width is approximately 960px) - i've created a separate issue #19 for it:
logo_screenshot2

**Comment by staaas** [comment]
@abitrolly you could have opened page in a private/incognito window to test the favicon.

This change LGTM.

File: filmfest/static/logo.svg (hunk 1 above)
Comment (abitrolly) [inline filmfest/static/logo.svg:36]: It is height and width of my Desktop. Inkscape records workspace parameters to .svg

**Pull request merged**